WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

```diff
@@ -28,33 +28,33 @@ Windows Server Insider Preview introduces an option to manually delimit the allo
 - Your cluster has fewer than six servers; or
 - Your cluster uses [parity](storage-spaces-fault-tolerance.md#parity) or [mirror-accelerated parity](storage-spaces-fault-tolerance.md#mirror-accelerated-parity) resiliency
 
-## Understand: Regular versus delimited allocation
+## Understand
 
-### Regular allocation
+### Review: regular allocation
 
 With regular three-way mirroring, the volume is divided into many small "slabs" that are copied three times and distributed evenly across every drive in every server in the cluster. For more details, read [this Deep Dive blog](https://blogs.technet.microsoft.com/filecab/2016/11/21/deep-dive-pool-in-spaces-direct/).
 
 ![regular-allocation](media/delimit-volume-allocation/regular-allocation.png)
 
 This default allocation maximizes parallel reads and writes, yielding better performance, and is appealing in its simplicity: every server is equally busy, every drive is equally full, and all volumes live and die together. Every volume is guaranteed to survive up to two concurrent failures, as [these examples](storage-spaces-fault-tolerance.md#examples) illustrate.
 
-However, with this allocation, no volume can survive three concurrent failures. If three servers fail at once, or if drives in three servers fail at once, every volume becomes inaccessible because at least *some* of its slabs were (with very high probability) allocated to the exact three drives or servers that failed.
+However, with this allocation, volumes can survive three concurrent failures. If three servers fail at once, or if drives in three servers fail at once, volumes become inaccessible because at least *some* slabs were (with very high probability) allocated to the exact three drives or servers that failed.
 
 In the example below, servers 1, 3, and 5 fail at the same time. Although many slabs have surviving copies, some do not:
```

Comment:
(P2) Voice principles say to use contractions - so recommend changing "do not" to don't.